enacts/maproom_utilities.py

```diff
@@ -0,0 +1,50 @@
+import dash_leaflet as dlf
```

Comment:
File name: mapping_utilities.py? Everything in this repo is for maprooms, e.g. calc.py and pingrid.py.

Reply:
I don't know if it will be useful outside of maprooms, but it doesn't cover all of maprooms, it's specific to the map-drawing part.

Reply:
more will come later in maproom_utilities.py